IM_calculation/IM/intensity_measures.py

```diff
@@ -5,7 +5,7 @@
 from IM_calculation.IM.Burks_Baker_2013_elastic_inelastic import Bilinear_Newmark_withTH
 
 DELTA_T = 0.005
-G = 981.0
+G = 981.0  # cm/s^2
```

Comment:
Sweet, can you modify AI below to use this G instead? I just realised it's doing a round-trip out of the file and back in again :rofl:

Reply:
Done.